src/main/java/org/opensearch/ad/model/AnomalyDetector.java

```diff
@@ -65,15 +65,14 @@
  *      in code rather than config it in anomaly-detection-state.json file.
  */
 public class AnomalyDetector implements Writeable, ToXContentObject {
-
     public static final String PARSE_FIELD_NAME = "AnomalyDetector";
     public static final NamedXContentRegistry.Entry XCONTENT_REGISTRY = new NamedXContentRegistry.Entry(
         AnomalyDetector.class,
         new ParseField(PARSE_FIELD_NAME),
         it -> parse(it)
     );
     public static final String NO_ID = "";
-    public static final String ANOMALY_DETECTORS_INDEX = ".opendistro-anomaly-detectors";
+    public static final String ANOMALY_DETECTORS_INDEX = "opendistro-anomaly-detectors";
```

Comment:
(Random thought, don't hold up PR approval for this)

Do we want to keep the opendistro name for backwards compatibility?  Although removing the dot (per 3.x) breaks that anyway, so we might want a new name.

Reply:
This is really good point. Do you think we should discuss this on an issue on SDK repo or if you have a better name I can replace it in this PR itself?
I can think of `opensearch-anomaly-detector`

Reply:
Actually, let me open an issue for this because I think it will require some discussion

Reply:
https://github.com/opensearch-project/opensearch-sdk-java/issues/241